README

```diff
@@ -1,11 +1,11 @@
 Pants Docker
-Version 0.1
```

Comment:
mmm I wouldn't necessarily version this as 1.0 since I don't think this is necessarily that big of an upgrade but I think we might just have different ideas about how versioning works lol

Reply:
I never feel confident versioning things... but for posterity: We agreed via Slack DM that 1.0 was appropriate in this case because the plugin is now live in production 😅